ci/build.sh

```diff
@@ -35,17 +39,32 @@ fi
 # with an invalid option error.
 export TASTY_NO_CREATE=true
 
+echo "::endgroup::"
+
+echo "::group::Install dependencies"
+
 # Install snapshot dependencies (since these will be cached globally and thus
 # can be reused during the sdist build step)
 $STACK build --only-snapshot $STACK_OPTS
 
+echo "::endgroup::"
+
+echo "::group::Build source distributions"
+
 # Test in a source distribution (see above)
 $STACK sdist lib/purescript-ast --tar-dir sdist-test/lib/purescript-ast
 tar -xzf sdist-test/lib/purescript-ast/purescript-ast-*.tar.gz -C sdist-test/lib/purescript-ast --strip-components=1
 $STACK sdist lib/purescript-cst --tar-dir sdist-test/lib/purescript-cst
 tar -xzf sdist-test/lib/purescript-cst/purescript-cst-*.tar.gz -C sdist-test/lib/purescript-cst --strip-components=1
 $STACK sdist . --tar-dir sdist-test;
 tar -xzf sdist-test/purescript-*.tar.gz -C sdist-test --strip-components=1
+
+echo "::endgroup::"
+
+echo "::group::Test in a source distribution"
```

Comment:
Same here, I think we don't need to draw attention to the fact that the tests are being run in a source distribution.